src/tracker/index.js

```diff
@@ -40,6 +40,16 @@ export default class Tracker {
       repo,
       accept: 'application/vnd.github.v3+json',
     };
+
+    this.initialize();
```

Comment:
The initialization should be executed by the caller not the constructor. See https://github.com/ambanum/OpenTermsArchive/pull/847